lib/jira/connect.js

```diff
@@ -45,12 +45,28 @@ module.exports = async (req, res) => {
             'pull_request',
           ],
           key: 'github-development-tool',
-          logoUrl: 'https://assets-cdn.github.com/images/modules/logos_page/GitHub-Mark.png',
           name: {
             value: 'GitHub',
           },
+          logoUrl: 'https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png',
           url: 'https://github.com',
         },
+        jiraDeploymentInfoProvider: {
```

Comment:
This needs to change to include a section specifying that it can push deployments and builds to Jira.